openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/Query.spec.js

```diff
@@ -0,0 +1,149 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { visitEntityDetailsPage } from '../../common/common';
+import { SEARCH_ENTITY_TABLE } from '../../constants/constants';
+
+describe('Add Query page', () => {
+  const table = SEARCH_ENTITY_TABLE.table_1;
+
+  const addQuery = ({ inputQuery, expectedErrorMessage }) => {
+    cy.get('[data-testid="add-placeholder-button"]')
+      .should('be.visible')
+      .click();
```

Comment:
can we update data testid of this button same as add query button id.

Reply:
done

Reply:
done